From 059247218728b5ac0efe533e4ee7ceb7236429d7 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Sun, 24 Oct 2021 18:46:45 +0300
Subject: [PATCH] [SW-60437] gaudi2: ppboot: raise the stack size

We have exhausted the stack size on Gaudi1 in Ppboot and raised it to
1K already some time ago.
It is just a short matter of time this will be needed for Gaudi2.
Do not wait for it, raise the stakc size now and sync with Gaudi1.

Change-Id: Iddefcf1e0a3a5bab5f685d90710266eccd35ce15
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/arch/gaudi2/arm/ppboot.lds.S b/arch/gaudi2/arm/ppboot.lds.S
index 662f0f5..57d6ad6 100644
--- a/arch/gaudi2/arm/ppboot.lds.S
+++ b/arch/gaudi2/arm/ppboot.lds.S
@@ -1,6 +1,6 @@
 /* SPDX-License-Identifier:     GPL-2.0+
  *
- * Copyright (C) 2020 HabanaLabs Ltd.
+ * Copyright (C) 2020-2021 HabanaLabs Ltd.
  */
 
 #include "common.h"
@@ -28,7 +28,7 @@
 	.stack : {
 		. = ALIGN(8);
 		__stack = . ;
-		. = . + 512 ;
+		. = . + 1024 ;
 		__stack_top = . ;
 	} > .sram
 
